packages/react/src/breadcrumbs/BreadcrumbsToggle.tsx

```diff
@@ -13,7 +13,7 @@ export function BreadcrumbsToggle({ onClick }: BreadcrumbsToggleProps) {
 			<Flex
 				as={BaseButton}
 				onClick={onClick}
-				aria-label="Expand list"
+				aria-label="Show all breadcrumbs"
```

Comment:
I don't know if "all" is the correct term, because that's not what this button is doing exactly…

Reply:
I'm happy to change it, this was just the suggestion in the audit. What I do like about it is the very clear statement that every breadcrumb will be shown after you press this button. Open to suggestions, mind you.

Reply:
Perhaps "show hidden breadcrumbs" or "show remaining breadcrumbs"? Probably worth asking these others

Reply:
Discussed with the team and opting for "Show all breadcrumbs".